packages/tool-cache/src/tool-cache.ts

```diff
@@ -23,30 +24,6 @@ export class HTTPError extends Error {
 const IS_WINDOWS = process.platform === 'win32'
 const userAgent = 'actions/tool-cache'
 
-// On load grab temp directory and cache directory and remove them from env (currently don't want to expose this)
```

Comment:
1. i moved this to a function otherwise can't override the location from unit tests (because loads on module import)
2. we shouldn't fallback to /Users/actions/temp - i got permissions issue when attempting to run setup-node tests locally. I added an assert that RUNNER_TEMP and RUNNER_TOOL_CACHE are defined.

Reply:
I copied the build output to setup-node to verify the unit tests are runnable with these changes